cmd/frcli/utils.go

```diff
@@ -5,14 +5,18 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/lightninglabs/faraday"
```

Comment:
nit: import order

Reply:
Oops :see_no_evil:.
Literally the only thing that annoys me in GoLand :joy: If they fix the auto import placement, this is the perfect Golang IDE for me :)